prototool.yaml

```diff
@@ -0,0 +1,51 @@
+# Paths to exclude when searching for Protobuf files.
```

Comment:
This looks like it could be added in a separate CL?

Reply:
Moved to #1527